src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Circle extends Shape {
+
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(Colors.valueOf(color));
```

Comment:
The `Colors.valueOf(color)` assumes that the color string is always a valid enum constant. Consider adding error handling to manage cases where the color might not be valid.